x/programs/examples/token.go

```diff
@@ -28,6 +28,11 @@ func NewToken(log logging.Logger, programBytes []byte, db state.Mutable, cfg *ru
 	}
 }
 
+type mintingRecipient struct {
+	To     [32]byte
```

Comment:
In the rust struct you use `recipient` as a field vs `To` shouldn't they be the same? I think it might read more logically as

```
type minter struct {
  To     [32]byte
  Amount int32
}